admin/server_info.php

```diff
@@ -100,6 +101,25 @@
 <?php } else { ?>
 <div class="phpinfo phpinfo-is-disabled"><?php echo ERROR_UNABLE_TO_DISPLAY_SERVER_INFORMATION; ?></div>
 <?php } ?>
+<h2 id="db-h2"><?php echo TITLE_DATABASE_VARIABLES . $system['db_version'] . ($system['mysql_strict_mode'] === true ? '<em> ' . TITLE_MYSQL_STRICT_MODE . '</em>' : ''); ?></h2>
+<table class="table" id="database-info">
+    <tr class="db-row">
+        <th class="db-name db-head db-info"><?php echo HEADING_DATABASE_VARIABLE; ?></th>
+        <th class="db-value db-head db-info"><?php echo HEADING_DATABASE_VALUE; ?></th>
+    </tr>
+    <?php
+    $show_variables = $db->Execute("SHOW VARIABLES");
+    while (!$show_variables->EOF) {
```

Comment:
Since this is now running on a version of Zen Cart that properly supports the use of `foreach` on the database queries, let's use that instead of `while/MoveNext()`.